.github/workflows/statediff.yml

```diff
@@ -0,0 +1,19 @@
+name: "Warn about consensus code changes"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    permissions:
+      pull-requests: write # For reading the PR and posting comment
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: orijtech/statediff@main
+        with:
+          roots: 'github.com/cosmos/cosmos-sdk/baseapp.BaseApp.DeliverTx,github.com/cosmos/cosmos-sdk/baseapp.BaseApp.BeginBlock,github.com/cosmos/cosmos-sdk/baseapp.BaseApp.EndBlock,github.com/cosmos/cosmos-sdk/baseapp.BaseApp.Commit'
```

Comment:
could you explain what this does?

Reply:
oh I read what this does. I don't think this solves the issue described above.

Reply:
@ebuchman could you please chime in too? @elias-orijtech implemented checking code reachable from the desired roots. Please chime in with suggestions for the use cases if it doesn't fit what he implemented. Thanks @tac0turtle @alexanderbez for the co-reviews!